resources/css/card.css

```diff
@@ -8,6 +8,10 @@
   font-size: theme('fontSize.lg');
 }
 
+.usercardaccounttype {
+  margin-left: 1em;
+}
```

Comment:
Please use a Tailwind class such as `ml-4` instead of adding a new class name.

Reply:
OK, done